icarusalg/Geometry/CMakeLists.txt

```diff
@@ -1,17 +1,20 @@
 
-cet_make(
-  SUBDIRS "details"
+file(GLOB src_files *.cxx details/*.cxx)
```

Comment:
OK for first pass: CMake best practice is to avoid `GLOB`bed library source lists wherever possible.

Reply:
I found the use of cet_make and art_make used because they do globbing.